app/views/backdate/edit.html.erb

```diff
@@ -0,0 +1,50 @@
+<% content_for :back_link, render_back_link(href: document_path(@edition.document)) %>
+<% content_for :title, t("backdate.edit.title") %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render_govspeak(t("backdate.edit.intro_paragraph")) %>
```

Comment:
We conventionally suffix things like this with '_govspeak' to make it clear to people like @benhazell how the content will be interpreted.

(https://github.com/alphagov/content-publisher/blob/master/docs/adr/0004-editing-microcopy.md#decision)